tests/store/artifact/test_http_artifact_repo.py

```diff
@@ -97,6 +98,13 @@ def test_log_artifact(http_artifact_repo, tmp_path, artifact_path, filename, exp
         with pytest.raises(Exception, match="request failed"):
             http_artifact_repo.log_artifact(file_path, artifact_path)
 
+    mpu_minimum_file_size = MLFLOW_MULTIPART_UPLOAD_MINIMUM_FILE_SIZE.get()
+    with mock.patch("os.path.getsize", return_value=mpu_minimum_file_size + 1), mock.patch.object(
+        http_artifact_repo, "_try_multipart_upload", return_value=200
+    ) as mock_mpu:
+        http_artifact_repo.log_artifact(file_path, artifact_path)
+        mock_mpu.assert_called_once()
+
```

Comment:
Is it possible to also test the abort scenario?

Reply:
i tested two more scenarios
- if underlying artifact store doesn't mpu
- if mpu is supported but failed, assert aborted

Reply:
thanks for the updates!